pkg/controller/resolvedofferings/region.go

```diff
@@ -0,0 +1,23 @@
+package resolvedofferings
+
+import (
+	"context"
+
+	v1 "github.com/acorn-io/runtime/pkg/apis/internal.acorn.io/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func AddDefaultRegion(ctx context.Context, c client.Client, app v1.AppInstance) error {
+	if app.GetRegion() == "" {
+		project := new(v1.ProjectInstance)
+		if err := c.Get(ctx, client.ObjectKey{Name: app.GetNamespace()}, project); err != nil {
+			return err
+		}
+
+		app.Status.ResolvedOfferings.Region = project.Status.DefaultRegion
```

Comment:
Does it matter if `project.Status.DefaultRegion` is `""`?

Reply:
I don't think it should ever be `""` since defaults are calculated before resolved offerings.

Reply:
The very first handler on the project router is the one that sets the default region. Since the project will always be created before any apps are, I think it is not possible for `project.Status.DefaultRegion` to be `""` here.

Reply:
Spoke about this in Slack, but it could happen. 

Grant explained that the metering API doesn't track regions though, so an empty region in the resolved offerings annotation is not impactful.

With that said, I'm cool with leaving this as is.